R/motifs.R

```diff
@@ -215,8 +216,13 @@ count_motifs <- function(graph, size = 3, cut.prob = rep(0, size)) {
 #' motifs(g, 3)
 #' count_motifs(g, 3)
 #' sample_motifs(g, 3)
-sample_motifs <- function(graph, size = 3, cut.prob = rep(0, size),
-                          sample.size = vcount(graph) / 10, sample = NULL) {
+sample_motifs <- function(
+  graph,
+  size = 3,
+  cut.prob = rep(0, size),
+  sample.size = NULL,
```

Comment:
I'm a bit uncomfortable about the default being hidden behind `NULL`, as the value is actually very relevant during usage, and this default value is completely arbitrary.

IMO the ideal solution would be to force users to provide either `sample.size` or `sample` explicitly instead of relying on the default. However, that is a breaking change not suitable for this release.

Reply:
could you please open an issue for this?

Reply:
Actually, the rationale is documented, the defaults are documented, and this will end up having a very low priority for a long time.

https://design.tidyverse.org/defaults-short-and-sweet.html#null-default

Reply:
We can change the interface to make arguments required, even in a lifecycle-friendly way. I think I misread this as a request to put back the inline default.

We could start with a message that is shown if the argument is omitted but would have been relevant.